src/index.js

```diff
@@ -0,0 +1,74 @@
+var inputElement = document.getElementById("commentInput");
```

Comment:
No need to commit your JS files. 

the repo should only include your ts source code.
whoever clones this repo will build the js files locally.